session-replay/Sources/DatadogSessionReplay/Recorder/Recorder.swift

```diff
@@ -21,9 +21,10 @@ internal class Recorder {
 
     convenience init() {
         self.init(
-            scheduler: MainThreadScheduler(interval: 0.2),
+            scheduler: MainThreadScheduler(interval: 0.25),
```

Comment:
🚧 Still using an arbitrary value here - the proper one will be explored later and / or be dynamic + configurable.